### DISCUSSION TITLE
What would be an approximate estimated time of run for DIAMOND with default parameters?

### DISCUSSION THREAD

**Discussion by Mayurk619** [bbuchfink/diamond#710] [Q&A] [answered]
I'm seeking an approximate runtime estimate for the Blastx alignment using diamond-linux64/2.1.6. The task involves aligning a 294 GB NCBI's nr database Index .dmnd file with 542 million reads mapping an 8 GB .fastq query file with 22.7 million reads. The server configuration includes 64 GB RAM, with each node having 24 cores. I'm limited to using a maximum of 120 cores across 10 nodes, with each node running 12 CPUs per task to generate the diamond alignment file.

I understand that the actual time can vary significantly based on various factors such as file size, number of reads, and other parameters. However, based on the paper I've read and the size of the reference database and query in my case, I'm estimating that it might take around 2 months. Would you say this is a reasonable estimate?

I do have a smaller test query dataset, but it's too small to provide a useful time estimate, as Diamond is optimized for handling millions of reads. Any guidance or insights you could provide would be greatly appreciated.

**Comment by bbuchfink** [comment, marked as answer]
This should take at most a couple of days, probably faster, depending on your sensitivity setting.

**Reply by Mayurk619** [reply]
Thank you. It took approximately 35 hours.
